goes16-l2-cmi.json

```diff
@@ -0,0 +1,472 @@
+            "roles": [
+                "data"
+            ]
+        },
+        "B05": {
+            "type": "application/netcdf",
+            "title": "Cloud and Moisture Imagery reflectance factor - ABI Band 05",
+            "roles": [
+                "data"
+            ]
+        },
+        "B06": {
+            "type": "application/netcdf",
+            "title": "Cloud and Moisture Imagery reflectance factor - ABI Band 06",
+            "roles": [
+                "data"
+            ]
+        },
+        "B07": {
+            "type": "application/netcdf",
+            "title": "Cloud and Moisture Imagery brightness temperature at top of atmosphere - ABI Band 07",
+            "roles": [
+                "data"
+            ]
+        },
+        "B08": {
+            "type": "application/netcdf",
+            "title": "Cloud and Moisture Imagery brightness temperature at top of atmosphere - ABI Band 08",
+            "roles": [
+                "data"
+            ]
+        },
+        "B09": {
+            "type": "application/netcdf",
+            "title": "Cloud and Moisture Imagery brightness temperature at top of atmosphere - ABI Band 09",
+            "roles": [
+                "data"
+            ]
+        },
+        "B10": {
+            "type": "application/netcdf",
+            "title": "Cloud and Moisture Imagery brightness temperature at top of atmosphere - ABI Band 10",
+            "roles": [
+                "data"
+            ]
+        },
+        "B11": {
+            "type": "application/netcdf",
+            "title": "Cloud and Moisture Imagery brightness temperature at top of atmosphere - ABI Band 11",
+            "roles": [
+                "data"
+            ]
+        },
+        "B12": {
+            "type": "application/netcdf",
+            "title": "Cloud and Moisture Imagery brightness temperature at top of atmosphere - ABI Band 12",
+            "roles": [
+                "data"
+            ]
+        },
+        "B13": {
+            "type": "application/netcdf",
+            "title": "Cloud and Moisture Imagery brightness temperature at top of atmosphere - ABI Band 13",
+            "roles": [
+                "data"
+            ]
+        },
+        "B14": {
+            "type": "application/netcdf",
+            "title": "Cloud and Moisture Imagery brightness temperature at top of atmosphere - ABI Band 14",
+            "roles": [
+                "data"
+            ]
+        },
+        "B15": {
+            "type": "application/netcdf",
+            "title": "Cloud and Moisture Imagery brightness temperature at top of atmosphere - ABI Band 15",
+            "roles": [
+                "data"
+            ]
+        },
+        "B16": {
+            "type": "application/netcdf",
+            "title": "Cloud and Moisture Imagery brightness temperature at top of atmosphere - ABI Band 16",
+            "roles": [
+                "data"
+            ]
+        }
```

Comment:
Great @marujore . Yes, we can consider a thumbnail for the items. Band 13 is a good candidate, considering its importance in meteorological applications. It will also be available in every observation, regardless of sunlight. I'll include it.